login/login.go

```diff
@@ -161,7 +166,7 @@ func (login *DeviceLogin) UpdateConfigFile() error {
 func NewLogin(args []string) error {
 	deviceLogin, err := NewDeviceLogin(args)
 	if err != nil {
-		return fmt.Errorf("failed to contact authentication service")
```

Comment:
Just checking: could the now left out information about where the error was raised not be good to have for debugging? Eg `failed to contact authentication service`, `failed parsing arguments` etc.

Reply:
That is a good point, however I think the returned error is xplanatory enough so that this text is not needed. If I am not mistaken this is something like:
```
./sda-cli login http://lalala
Error: Get "http://lalala/info": dial tcp: lookup lalala: no such host
```

Reply:
I also agree with Malin. I'd consider it best practice to include source error messages unless there is a security concern (e.g. risk of e.g. information leak).

Since this is on the client, no such risk.

(I also note that e.g. the "no such host" error shown above can be vital information point in debugging, e.g. in an enterprise system with broken DNS, or of course "no route to host" or "connection refused" would also be important parts in figuring out what goes wrong. I've actually heard that there are people that do not like starting their debugging with `strace` to figure out what system calls are made and what they return.)

Reply:
I changed back the extra error messages and added the actual err with them.